Jenkinsfile

```diff
@@ -39,9 +39,33 @@ dockerTemplate {
         }
       }
 
+      stage('Test') {
+
+        // Spawn the image just created
+        mavenNode(mavenImage:image) {
+          container('maven') {
+            echo 'Container spawned successfully'
+            sh 'cat /etc/redhat-release'
+            sh 'hostname'
+            sh 'java -version'
+            sh 'mvn -version'
+            sh 'git --version'
+
+            // I've no idea where the canonical source for booster apps are, so
+            // let's clone a fork instead
+            sh 'git clone https://github.com/jaseemabid/vertxapp'
+            dir('vertxapp') {
+              sh 'pwd'
+              sh 'mvn clean -B -e -U package -Dmaven.test.skip=false -P openshift'
+            }
+          }
+        }
+      }
+
       if (utils.isCI()) {
         stage('Notify the PR') {
-          if (!env.CHANGE_ID) {
+          def pr = env.CHANGE_ID
```

Comment:
How is this related to the commit?  Also, IMHO, this doesn't improve readability

Reply:
@sthaha This is a regression I introduced earlier. The next line actually refers to this variable. The previous version would actually fail with undefined variable error.